packages/api/src/post-build.js

```diff
@@ -31,6 +32,10 @@ class PostBuildHandler {
     this.client = client;
   }
 
+  static async addBuild (testCases, buildInfo, client, collectionName) {
+    await addBuild(testCases, buildInfo, client, collectionName);
+  }
```

Comment:
Is there a reason for this static method if it's just calling another function?

Reply:
Yes, it's much simpler to stub addBuild this way. Alternatively, I believe, we could have addBuild `export {addBuild}`. This would allow us to stub the original function, though then every time we used it we have to do something like `addBuildModule.addBuild`. @bcoe would you prefer to keep it this way or should I change it?

Here's an article I found about exactly this situation: https://medium.com/@minaluke/how-to-stub-spy-a-default-exported-function-a2dc1b580a6b

Reply:
I agree with @azizsonawalla that the approach we took of encapsulating the method in a `static` method reads a little odd ... I don't have a strong opinion though, I like how easy it makes our mocking.

If @azizsonawalla makes a case for it, I think a reasonable alternative would be that we switch to two exports like the article you shared suggests, so:

```
module.exports = {
  addBuild,
  PostBuildHandler
}
```

I don't think we should pull `proxyquire` into the project, as we're already using `nock` and `sinon`.

Reply:
I just went ahead and tried having `add-build` export `{addBuild}` instead of `addBuild`, and it took a bit of refactoring but it works now without that little function.